routes/bubble.js

```diff
@@ -1,15 +1,34 @@
 var express = require('express');
 var router = express.Router();
+const Bubble = require('../models/Bubbles')
 const sequelize = require("../models/index")
+var randomstring = require("randomstring");
+
 
 //to create a data base based on my schemas
 sequelize
 .sync()
-.then(res => console.log(res)
-.catch(err=>console.log(err)))
+.then(res => console.log(res))
+
 
 router.get("/",(req,res)=>{
     res.send("test")
 })
 
+router.post("/create",(req,res)=>{
+    let data = req.body;
+    let bubbleName = data.name;
+    let permHash = randomstring.generate({
+        length: data.name.length,
+        charset: data.name
+      });
+    Bubble.create({

```

Comment:
Keep in mind that Sequelize is a promise-based package. Always include .then() and .catch to any promise based method and include an appropriate response.

Reply:
bubblesModel.create()